core/node/eth_sender/src/eth_tx_manager.rs

```diff
@@ -508,15 +547,6 @@ impl EthTxManager {
             .track_eth_tx_metrics(storage, BlockL1Stage::Mined, tx)
             .await;
 
-        if let Some(predicted_gas_cost) = tx.predicted_gas_cost {
-            if gas_used > U256::from(predicted_gas_cost) {
-                tracing::error!(
-                    "Predicted gas {predicted_gas_cost} lower than used gas {gas_used} for tx {:?} {}",
-                    tx.tx_type,
-                    tx.id
-                );
-            }
-        }
```

Comment:
any reason to remove it?

Reply:
yes, and it was slightly harder to get. 
But tx, will fail. you are right